contracts/Core/Parameters.sol

```diff
@@ -14,9 +14,9 @@ contract Parameters is ACL {
 
     uint256 public minStake = 100 * (10**18);
     uint256 public withdrawLockPeriod = 1;
-    uint256 public maxAltBlocks = 5;
-    uint256 public epochLength = 300;
-    uint256 public numStates = 4;
+    uint8 public maxAltBlocks = 5;
+    uint32 public epochLength = 300;
+    uint32 public numStates = 4;
```

Comment:
This can be uint8. Unlikely we will go above numStates = 255

Reply:
this pr doesn't completely optimise everything. futher optimisations can be done in future pr.